waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java

```diff
@@ -194,27 +194,27 @@ public void shutdown() {
 
   @Override
   @Loggable(value = Loggable.DEBUG, skipResult = true, name = INVOCATION_LOG_NAME)
-  public String getMetaConf(String key) throws MetaException, TException {
+  public String getMetaConf(String key) throws TException {
```

Comment:
OK, so the exception signatures on this class come from implementing `org.apache.hadoop.hive.metastore.api.ThriftHiveMetastore` which has the `MetaException` even thought it extends `TException`. Same for the methods below. We could remove them but I think I'd actually prefer to keep them as they're more self-documenting which I assume is one of the reasons they were kept in the original interface. I'd rather we stick with what the class we're implementing does, even if it's not perfect.